src/alfasim_score/converter/alfacase/convert_alfacase.py

```diff
@@ -59,10 +80,96 @@ def _convert_annulus(self) -> AnnulusDescription:
     def _convert_formation(self) -> AnnulusDescription:
         return FormationDescription(reference_y_coordinate=Scalar(0.0, "m", "length"))
 
+    def _convert_casing_list(self) -> List[CasingSectionDescription]:
+        """Create the description for the casings."""
+        casing_sections = []
+        i = 1
+        for data in self.score_input.read_casings():
+            for section in data["sections"]:
+                casing_sections.append(
+                    CasingSectionDescription(
+                        name=f"{data['function'].value}_{data['type'].value}_{i}",
```

Comment:
I'd rather name is:
```
for section_id, section in enumerate(data["sections"]):
 name=f"{data['function'].value}_`casing_{section_id}",
```

Following what you have done in ALFAwtin, thus, we would have:
- conductor_casing_0
- surface_casing_0
- production_casing_0
- production_casing_1
- production_casing_2

Reply:
It makes sense. I'm just gonna start at 1 once things seem to start at 1 in the SCORE file.

Reply:
changed